wiki/Wiki_Contribution_Guide/en.md

```diff
@@ -0,0 +1,16 @@
+# Titanic! Wiki Contribution Guide
+
+Thanks for your interest in helping out to improve the Titanic! wiki. We really appreciate it.
+This is a step-by-step guide on how you can help make changes to the wiki.
+
+To begin with you will need to have a basic knowledge of GitHub or [git](https://git-scm.com/). It's okay if you get confused, you can always ask in the [Titanic! Discord server](https://discord.gg/qupv72e7YH) in the `#wiki` channel.
+
+## Editing the article
+
+We recommend reading the [osu! wiki's contribution guide](https://osu.ppy.sh/wiki/en/osu!_wiki/Contribution_guide) since a lot of that will also apply to the Titanic wiki.
+
+## Review
```

Comment:
I might try to write smth here if there is some context, right now i'm just currently confused.

At the same time the Review should be something related to translations if i'm correct.

Reply:
Is this meant to be a section teaching you how to submit a review on pr's?
I mean, that could be helpful yeah